ctrlf.el

```diff
@@ -884,12 +899,20 @@ change the search style if already in a search."
     (unless preserve
       (setq ctrlf--style style))
     (if ctrlf--active-p
-        (if (and (not inhibit-history)
-                 (string-empty-p (field-string (point-max))))
-            (previous-history-element 1)
-          (ctrlf-next-match))
+        (cond
+         ;; Continue search with `arg'.
+         ;; TODO: Is there a better way to replace the minibuffer contents?
```

Comment:
Just calling out this question again 😂

Reply:
If given `arg` I just want this function to replace any existing input with it.

Reply:
Maybe, but `delete-minibuffer-contents` followed by `insert` looks pretty great to me.

Reply:
Cool!